code/crates/core-driver/src/driver.rs

```diff
@@ -227,6 +232,11 @@ where
         &self.polka_certificates
     }
 
+    /// Get the round certificate for the given round.
```

Comment:
```suggestion
    /// Get the round certificate for the current round.
```
Since the round is not passed here?